<launch>
  <node  name="demo_3d_nav" pkg="3d_nav_executive" type="demo_3dnav" output="screen" > 
    <param name="arm_positions_filename" value="$(find 3d_nav_executive)/config/arm_poses.txt" />
    <param name="goal_positions_filename" value="$(find 3d_nav_executive)/config/goals.txt" />
  </node>

</launch>
